lib/assignment_manager.rb

```diff
@@ -59,6 +59,16 @@ def claim_assignment(claimed_assignment)
     end
   end
 
+  def self.check_wiki_edu_discouraged_article(title:)
+    category = Category.find_by('article_titles LIKE ? AND name = ?', "%#{title}%",
```

Comment:
It looks like this is looking for a Category named 'Wiki_Edu_Discouraged_Articles', and also matching based on article title?